src/content/docs/en/guides/server-side-rendering.mdx

```diff
@@ -230,4 +230,100 @@ if (!product) {
 ### Server Endpoints
 
 A server endpoint, also known as an **API route**, is a special function exported from a `.js` or `.ts` file within the `src/pages/` folder. 
-The function takes an [endpoint context](/en/reference/api-reference/#endpoint-context) and returns a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response). A powerful feature of SSR, API routes are able to securely execute code on the server. To learn more, see our [Endpoints Guide](/en/core-concepts/endpoints/#server-endpoints-api-routes).
\ No newline at end of file
+The function takes an [endpoint context](/en/reference/api-reference/#endpoint-context) and returns a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response). A powerful feature of SSR, API routes are able to securely execute code on the server. To learn more, see our [Endpoints Guide](/en/core-concepts/endpoints/#server-endpoints-api-routes).
+
+### Streaming
+
+Since browsers support HTTP streaming, a document will be broken up into chunks, sent over the network in order, and rendered on the page in that order.
+
+Astro additionally uses streaming to help "unblock" data fetching. Before this feature, Astro would wait until all data finished loading before sending HTML. Now, Astro will render and send components over the network until it finds one that `await`s a Promise. It will wait for the component's Promises to resolve, then render and send the component over the network. 
```

Comment:
I would probably describe how this works "without" streaming rather than saying "before". I don't think a before/after is very interesting to anyone who wasn't around in the "before" and it becomes increasingly irrelevant as time goes on. But the difference in approaches in the approaches themselves is eternal.